src/RadixDlt.NetworkGateway.PostgresIntegration/Services/EntityStateQuerier.cs

```diff
@@ -527,8 +535,10 @@ LIMIT 1
         return result;
     }
 
-    private async Task<GatewayModel.FungibleResourcesCollection> GetFungiblesSlice(long entityId, int offset, int limit, GatewayModel.LedgerState ledgerState,
-        CancellationToken token)
+    /// <summary>
+    /// Returns a paginable resource collection with total (aggregated) balance per resource.
+    /// </summary>
+    private async Task<GatewayModel.FungibleResourcesCollection> GetFungiblesSlice(long entityId, int offset, int limit, GatewayModel.LedgerState ledgerState, CancellationToken token)
```

Comment:
can we rename it to `GetAggregatedFungibleSlice` so we could easily distinguish it from `GetFungiblesSlicePerVault`?

Reply:
Renamed to *PerResource to match *PerVault scheme.